vnext/ReactUWP/Views/ReactControl.cpp

```diff
@@ -338,7 +338,7 @@ void ReactControl::ShowDeveloperMenu() {
       L"    <Button HorizontalAlignment='Stretch' x:Name='Reload'>Reload Javascript (TBD) </Button>"
       L"    <Button HorizontalAlignment='Stretch' x:Name='RemoteDebug'></Button>"
       L"    <Button HorizontalAlignment='Stretch' x:Name='LiveReload'>Enable Live Reload (TBD) </Button>"
-      L"    <Button HorizontalAlignment='Stretch' x:Name='Inspector'>Show Inspector (TBD) </Button>"
+      L"    <Button HorizontalAlignment='Stretch' x:Name='Inspector'>Show Inspector</Button>"
```

Comment:
>Show Inspector [](start = 69, length = 14)

The text "Show Inspector" would ideally toggle between "Show Inspector" and "Hide Inspector".  Can you query the current show/hide state and update the text as appropriate?

Reply:
This behavior is consistent with iOS/Android. Do we want to deviate from it?

Reply:
Yes.  It feels wrong to say "Show XYZ" if XYZ is already showing.  I would rather the text say "Toggle Inspector" if this will be difficult to implement.

Reply:
Oh yeah, didn't notice the label. I'll change the label to Toggle Inspector to match iOS/Android.